content/news/181-lbry-is-not-an-app.md

```diff
@@ -0,0 +1,20 @@
+---
+author: sean-yesmunt
+title: 'LBRY Is Not An App'
+date: ''
+cover: '.png'
+---
+
+## LBRY Is Not An App
+
+Oh jeez, another post about how LBRY is a protocol. I wanted to write this because it seems like an area that is confusing for some people (my parents included). LBRY is not an app, even though I can see why you might think that. When I first discovered LBRY I thought it was just a desktop app, however, the real technology is what powers it, the LBRY protocol. 
+
+Currently there are three applications that use the LBRY protocol: the desktop/android app, and spee.ch. All three of these can read from and write data to the LBRY blockchain. Most users browsing the LBRY network through one of these apps (or a future one that hasn't been created yet) don't care about a blockchain in the underbellies of the app, the same way someone watching YouTube doesn't care how http works. [The technology linking the blockchain to these apps _is_ LBRY](https://github.com/lbryio/lbry). Any number of apps can be built on top of LBRY, and each one can use all or a subset of the data stored on the network. 
```

Comment:
> Currently there are three applications that use the LBRY protocol: the desktop/android app, and spee.ch.

The slash makes it seem like the apps are either or, not standalone things.

> the desktop and Android apps, and spee.ch

makes it more apparent.